kits/rosie/arm.hpp

```diff
@@ -0,0 +1,31 @@
+#include "robot_model.hpp"
+#include "trajectory.hpp"
+
+using namespace hebi;
+
+class GripperArm{
```

Comment:
I'm used to a class going into a file with the same name -- e.g., `gripper_arm.hpp` here.

I'm not sure how useful this data structure is for this one demo, but we might end up reusing it in other files.  I think the utility will go down after the arm API is finished/moved into the C++ API.

Reply:
yeah I could take it out, I don't think its super necessary either